web/src/ui_util.ts

```diff
@@ -22,14 +22,39 @@ export function place_caret_at_end(el: HTMLElement): void {
     }
 }
 
-export function replace_emoji_with_text($element: JQuery): void {
+export function replace_emoji_name_with_unicode_emoji_code($element: JQuery): void {
     $element
         .find(".emoji")
         .text(function () {
+            // We don't have corresponding unicode for a custom emoji.
             if ($(this).is("img")) {
                 return $(this).attr("alt") ?? "";
             }
-            return $(this).text();
+
+            // Emojis have a class with the emoji code next to them e.g. ".emoji-1f419".
+            // The code "1f419" represents an octopus.
+            const emoji_class_with_code: string | undefined = $(this).attr("class");
```

Comment:
This variable name is misleading, since the class attribute is string containing a space-separated list of multiple classes.